doc/content/reference/interop-repository/_index.md

```diff
@@ -42,38 +46,68 @@ A configuration like this would make Join Server requests for JoinEUIs starting
 The Join Server configuration provides means to configure how the components interact with the Join Server. The configuration supports multiple options:
 
 ```yml
-fqdn: 'thethings.example'                 # FQDN of the Join Server, if unset, it is resolved via LoRa Alliance DNS
-port: 12345                               # port to connect at, if unset, 443 is used
-protocol: 'BI1.0'                         # protocol to use - one of BI1.0 or BI1.1
-paths:                                    # custom URI paths to use for various requests, if unset, the FQDN is used
-  join: 'some/path'                       # the URI path to use for JoinReq
-  rejoin: 'some/other/path'               # the URI path to use for RejoinReq
-  app-s-key: 'other/path'                 # the URI path to use for AppSKeyReq
-  home-ns: 'somepath'                     # the URI path to use for HomeNSReq
-tls:                                      # TLS configuration to use
-  root-ca: 'path/to/clientca.pem'         # path to client CA
-  certificate: 'path/to/clientcert.pem'   # path to client TLS certificate
-  key: 'path/to/clientkey.pem'            # path to client TLS key
-headers:                                  # HTTP headers to send, defined as key-value map
-  SomeHeader: 'SomeValue'
+scheme: 'https'                          # URL scheme. Defaults to https
+fqdn: 'thethings.example'                # FQDN of the Join Server
+port: 12345                              # port to connect at. Defaults to 443
+protocol: 'BI1.0'                        # Backend Interfaces protocol to use (one of BI1.0 or BI1.1)
+paths:                                   # custom URI paths to use for various requests. Defaults to /
+  join: 'some/path'                      # the URI path to use for JoinReq
+  rejoin: 'some/other/path'              # the URI path to use for RejoinReq
+  app-s-key: 'other/path'                # the URI path to use for AppSKeyReq
+  home-ns: 'somepath'                    # the URI path to use for HomeNSReq
+sender-ns-id: '1122334455667788'         # Backend Interfaces 1.1 NSID
+basic-auth:                              # HTTP Basic Authentication (optional)
+  username: 'user'                       # HTTP Basic username
+  password: 'secret'                     # HTTP Basic password
+tls:                                     # TLS configuration to use (optional)
+  root-ca: 'path/to/clientca.pem'        # path to client CA
+  certificate: 'path/to/clientcert.pem'  # path to client TLS certificate
+  key: 'path/to/clientkey.pem'           # path to client TLS key
+headers:                                 # HTTP headers to send, defined as key-value map
+  Some-Header: 'SomeValue'
+```
+
+### The Things Join Server
```

Comment:
```suggestion
## The Things Join Server
```

Reply:
It's better to have this included in the right hand navigation, so I think you should remove one "#". I would also change this title to something else, perhaps "Example: The Things Join Server", "Interoperability with The Things Join Server" or whatever you can think of, because we already have a doc named "The Things Join Server" ([here](https://www.thethingsindustries.com/docs/getting-started/cloud-hosted/tti-join-server/)) and search results for that phrase can start to look not so nice. Also, please see if there's a need for some cross-linking between this section and [this](https://www.thethingsindustries.com/docs/getting-started/cloud-hosted/tti-join-server/)